misc/config_tools/static_allocators/bdf.py

```diff
@@ -119,7 +119,25 @@ def create_device_node(allocation_etree, vm_id, devdict):
         if common.get_node(f"./func", dev_node) is None:
             common.append_node(f"./func", f"{bdf.func:#04x}", dev_node)
 
+def create_gpu_sbdf(board_etree, allocation_etree):
+    bus = "0x0"
+    hv_node = common.get_node(f"/acrn-config/hv", allocation_etree)
+    if hv_node is None:
+        hv_node = common.append_node("/acrn-config/hv", None, allocation_etree)
+    misc_cfg_node = common.get_node(f"./MISC_CFG", hv_node)
+    if misc_cfg_node is None:
+        misc_cfg_node = common.append_node("./MISC_CFG", None, hv_node)
+    device_node = common.get_node(f"//bus[@type='pci' and @address='{bus}']/device[vendor='0x8086' and class='0x030000']", board_etree)
```

Comment:
There is the method `append_node` in `common.py` which can help you create those ancestor nodes with one single call.